index.bs

```diff
@@ -203,7 +203,7 @@ permission-related algorithms and types are defined as follows:
   Given a {{FileSystemPermissionDescriptor}} |desc| and a {{PermissionStatus}} |status|,
   run these steps:
 
-  1. Run the [=boolean permission query algorithm=] on |desc| and |status|.
+  1. Run the <a spec=permissions>default permission query algorithm</a> on |desc| and |status|.
```

Comment:
Do these need the explicit spec= to disambiguate, or does it link fine without that as well (in which case it could also go back to the `[= ... =]` shorthand)?

Reply:
Ah good catch. The shorthand works